### PR TITLE
Add libsecret to make auto login work again

### PR DESCRIPTION
**Pull request by snaggen**
It seems that the runtime used doesn't contain libsecret, which is needed for auto login.

### CODE REVIEW

**Comment by snaggen** [comment]
Asking for review, since I don't know if the problem is with the runtime or if we should add it here

**Comment by TingPing** [comment]
Skype is just an Electron app and Electron uses gnome-keyring not libsecret afaik?

**Comment by snaggen** [comment]
Skype switched from gnome-keyring to use libsecret

File: com.skype.Client.json [hunk 1]
```diff
@@ -31,6 +31,24 @@
     ],
     "modules": [
         {
+            "name": "libsecret",
```
Comment (TingPing) [inline com.skype.Client.json]: Needs cleanups for `/include`, `/lib/pkgconfig`, etc

File: com.skype.Client.json [hunk 2]
```diff
@@ -31,6 +31,24 @@
...
+            "config-opts": ["--disable-gtk-doc",
```
Comment (TingPing) [inline com.skype.Client.json]: `--disable-static`

File: com.skype.Client.json [hunk 3]
```diff
@@ -29,6 +29,11 @@
         "--talk-name=org.kde.StatusNotifierWatcher",
         "--extra-data=skypeforlinux-64.deb:99ed59bfc5521f643071694a23f2387dac016038776d48ab03165c2e35c5c5e4:53937734::https://repo.skype.com/latest/skypeforlinux-64.deb"
     ],
+    "cleanup": ["/include", "/lib/pkgconfig",
```
Comment (TingPing) [inline com.skype.Client.json]: We can do better than generic copy pastes. Some of these are random and don't apply to this.

Comment (snaggen) [reply com.skype.Client.json]: Not sure what the harm is... but if you say so...

Comment (TingPing) [reply com.skype.Client.json]: Flatpak manifests are already copy-paste central and I have to deal with that, by keeping cleanups with their modules there is no global state that can be lost easily, and localized cleanups can be minimal and accurate.

Comment (TingPing) [reply com.skype.Client.json]: Ah, and my point exactly your config-opts have copy paste garbage too...

**Comment by snaggen** [comment]
Where is the face palm emoji when it is needed... Yes, build flags copy paste is not that good. I apologize and promise to be more careful and less snarky next time.

**Comment by TingPing** [comment]
It happens. Sorry if I came off as pedantic but we certainly don't want Flathub to be the place with low quality packages and prove all the anti-bundler folks right =).

**Pull request merged**